backend/src/routes/post.ts

```diff
@@ -0,0 +1,61 @@
+import Router from '@koa/router';
+import Post from '../models/post';
+import mongodb from 'mongodb';
+
+const router = new Router({
+	prefix: '/posts',
+});
+
+router.get('/', async (ctx, next) => {
+	ctx.body = await Post.find();
+});
+
+router.get('/:postID', async (ctx, next) => {
+	const post = await Post.findOneBy(mongodb.ObjectId(ctx.params.postID));
+
+	if (post === null) {
+		ctx.status = 404;
+		return;
+	}
+	ctx.body = post;
+});
+router.post('/:netID/:body/:musicURL', async (ctx, next) => {
+	const post = new Post();
+	post.id = new mongodb.ObjectId();
+	post.netID = ctx.params.netID;
+	post.numLikes = 0;
+	post.bodyText = ctx.params.body;
+	post.musicURL = ctx.params.musicURL;
```

Comment:
I think we should get these two values from the POST body and not from the URL. I recommend installing [`koa-bodyparser`](https://www.npmjs.com/package/koa-bodyparser), `app.use`'ing it, and getting the body and musicURL from `ctx.request.body`.